Tests/JWTKitTests/X5CTests.swift

```diff
@@ -165,6 +166,48 @@ final class X5CTests: XCTestCase {
         XCTAssertEqual(data.appAppleId, 1234)
         XCTAssertEqual(data.environment, "Sandbox")
     }
+
+    func testSigningWithX5CChain() async throws {
```

Comment:
We should test a chain with a cert that isn't verified by the node above to make sure it throws an error